packages/ui/src/components/Command/AiCommand.tsx

```diff
@@ -483,7 +483,7 @@ const AiCommand = () => {
                     search ? 'opacity-100' : 'opacity-0'
                   }`}
                 >
-                  <span className="text-scale-1100">Submit message</span>
+                  <button className="text-scale-1100">Submit message</button>
```

Comment:
If we are changing this into a button, can we add the `onClick` handler as well?

Reply:
It depends on what we want it to do on being clicked. I had added that change because I thought of positioning the button relative to the input container so that the overlapping situation can be avoided but then you shared a pr which was helpful in resolving the issue,the button did not pose any issue to the alignment either on my local machine so I did not change it ,should I revert it back to span?

Reply:
Got it. Let's revert back to `<span>` for now and we can consider the button in the future.